paasta_tools/long_running_service_tools.py

```diff
@@ -329,7 +332,7 @@ def get_autoscaling_params(self) -> AutoscalingParamsDict:
         default_params: AutoscalingParamsDict = {
             "metrics_provider": "mesos_cpu",
             "decision_policy": "proportional",
-            "setpoint": 0.8,
+            "setpoint": DEFAULT_AUTOSCALING_SETPOINT,
```

Comment:
this is where I would do the change I proposed in https://github.com/Yelp/paasta/pull/3027#discussion_r569883664, you may or may not want to do it in this commit